nansat/vrt.py

```diff
@@ -1496,12 +1494,7 @@ def transform_points(self, col_vector, row_vector, dst2src=0,
         lonlat = transformer.TransformPoints(dst2src, xy)[0]
```

Comment:
what is lonlat? Could you add a comment, please? (Description of structure or something)

Reply:
Fixed in 96fbd90